_includes/footer.html

```diff
@@ -1,30 +1,169 @@
 <footer class="usa-footer site-footer" role="contentinfo">
-  <div class="footer-section-bottom bg-base-lighter">
-    <div class="grid-container">
-      <div class="grid-row padding-3">
-        <div class="tablet:grid-col">
-          <div class="logo-links">
-            <a class="footer-logo media_link" href="https://cloud.gov">
-              {% image_with_class "./_img/logo-img.png" "width-10 height-auto" "Agency logo" %}
-            </a>
-          </div>
+    <div class="footer-section-bottom usa-footer__primary-section bg-accent-warm-light border-top border-accent-warm-dark">
```

Comment:
Would love to see this footer populated by a yaml list in the _data folder rather than hardcoded values.